packages/vuetify/build.config.mts

```diff
@@ -0,0 +1,94 @@
+import { defineBuildConfig } from 'unbuild'
```

Comment:
Why are you using a separate build process for this?

Reply:
We must split client and node builds, I can move it to rollup but I like unbuild.

Reply:
done: 
- added some jsdocs to rollup config files to have auto complettion and suggestions in the ide
- removed `unbuild` and moved `unplugin/*.ts`  build to Rollup
- fixed also the lint script: added rootDir and custom exclude in `tsconfig.check.json`